### ISSUE TITLE
Please update your mod to 1.20

### ISSUE THREAD

**Issue by PedroLVA** [UltimateBoomer/smoothboot-fabric#54]
I believe that your mod fixed the chunk-crossing gigantic lagspike issue for me and for a lot of people
Its amazing! So please update it

**Comment by EinfxchFinn** [comment]
Smooth Boot is no longer needed in 1.20
See https://modrinth.com/mod/smoothboot-fabric

**Comment by Kennybust** [comment]
> Smooth Boot is no longer needed in 1.20 See https://modrinth.com/mod/smoothboot-fabric

Wrong, Mojang only fixed Minecraft start up time issue, but not Minecraft CPU bottleneck and chunk generation, loading, which smooth boot fixes i believe by thread priority management (sorry if i'm wrong), here are some screenshots:
Without Smooth Boot (1.19.4)
![without](https://github.com/UltimateBoomer/mc-smoothboot/assets/80032036/c0dc0fc0-d236-4597-a797-01b07ec3b2e1)
With Smooth Boot (1.19.4)
![With](https://github.com/UltimateBoomer/mc-smoothboot/assets/80032036/d13f0697-2dfb-41ac-b205-7b9b69916c4a)

It also fixes chunk generation stutter in 1.20.1 as well if i change version support in config file:
With smooth boot (1.20.1)
![with 1 20 1](https://github.com/UltimateBoomer/mc-smoothboot/assets/80032036/f54d9546-902f-4046-817f-1aa0f0185e27)
Without smooth boot (1.20.1)
![Without 1 20 1](https://github.com/UltimateBoomer/mc-smoothboot/assets/80032036/9784a784-368d-45b6-8874-6b0bc1b4f9f6)

**Comment by PedroLVA** [comment]
> > Smooth Boot is no longer needed in 1.20 See https://modrinth.com/mod/smoothboot-fabric
> 
> Wrong, Mojang only fixed Minecraft start up time issue, but not Minecraft CPU bottleneck and chunk generation, loading, which smooth boot fixes i believe by thread priority management (sorry if i'm wrong), here are some screenshots: Without Smooth Boot (1.19.4) ![without](https://user-images.githubusercontent.com/80032036/246169377-c0dc0fc0-d236-4597-a797-01b07ec3b2e1.png) With Smooth Boot (1.19.4) ![With](https://user-images.githubusercontent.com/80032036/246169472-d13f0697-2dfb-41ac-b205-7b9b69916c4a.png)
> 
> It also fixes chunk generation stutter in 1.20.1 as well if i change version support in config file: With smooth boot (1.20.1) ![with 1 20 1](https://user-images.githubusercontent.com/80032036/246171411-f54d9546-902f-4046-817f-1aa0f0185e27.png) Without smooth boot (1.20.1) ![Without 1 20 1](https://user-images.githubusercontent.com/80032036/246171130-9784a784-368d-45b6-8874-6b0bc1b4f9f6.png)

How can you run smooth boot in 1.20.1? You said to change the supported version in the config file, how do you do that?

**Comment by Kennybust** [comment]
@PedroLVA You download the latest smooth boot mod version, than open it with archiver like 7zip, and edit fabric.mod.json file
![1](https://github.com/UltimateBoomer/mc-smoothboot/assets/80032036/cb477153-2f6b-4dbd-bb83-d6c1a57101e7)
than add this ">=" next to 1.19.4 in this field
![2](https://github.com/UltimateBoomer/mc-smoothboot/assets/80032036/ee7ccb12-91a8-4c23-b0fb-6fccd368da9d)
will look like this 
![3](https://github.com/UltimateBoomer/mc-smoothboot/assets/80032036/884cd8f5-75e6-47f3-915e-7f78da007d2f)
than save it in archive, and add mod to your mods folder

**Comment by PedroLVA** [comment]
@Kennybust Thank you so much, you're a life saver!

**Comment by notabirb1** [comment]
@Kennybust you can also use [fabric dependency overrides](https://fabricmc.net/wiki/tutorial:dependency_overrides) if you don't want to edit the mod's files.

**Comment by safesaffawa** [comment]
I made a version 1.20 that only changed the version 
https://github.com/safesaffawa/mc-smoothboot-1.20/releases/tag/1.19-1.7.1

**Comment by notabirb1** [comment]
There's an [updated fork](https://modrinth.com/mod/threadtweak) now to port the thread priority changes.

**Comment by Felix14-v2** [comment]
My two cents, with/without
![image-10](https://github.com/UltimateBoomer/mc-smoothboot/assets/75726196/36a8baed-ce66-4994-b8be-906df2e6a272)
![image-9](https://github.com/UltimateBoomer/mc-smoothboot/assets/75726196/f97e0469-0987-483a-9123-cde47e70feac)

**Comment by intergrav** [comment]
Thought I might chip in here, @getchoo and I have forked the mod and published it to Modrinth and will be maintaining it for future versions (with the creator's permission of course) Feel free to check it out!
- Modrinth project page: https://modrinth.com/mod/threadtweak
- GitHub repository: https://github.com/getchoo/threadtweak

**Comment by matzar24** [comment]
Will be there 1.20 forge?

**Comment by Apollonu** [comment]
> Will be there 1.20 forge?

https://www.curseforge.com/minecraft/mc-mods/threadtweak-reforged
https://www.curseforge.com/minecraft/mc-mods/smooth-boot-reloaded
